src/components/shared/Exercises/ExerComplete.tsx

```diff
@@ -0,0 +1,20 @@
+import '../../../styles/Exercises/ExerComplete.scss';
+import encouragingTurtle from '../../../assets/EncouragingTurtle-Solo.svg';
+
+interface ExerCompleteInputs {
+  exerciseName: string;
+}
+
+function ExerComplete({ exerciseName }: ExerCompleteInputs): JSX.Element {
```

Comment:
This is a new thing, and I didn't ask for you to do it earlier, but I think it'll make it easier going forward if you have this take a void function that takes no parameters, and call that on the exercise complete button